README.md

```diff
@@ -557,15 +557,19 @@ Note that the flattener does not deal with arrays at this time.
 
 By default, the fluentd elasticsearch plugin does not emit records with a _id field, leaving it to Elasticsearch to generate a unique _id as the record is indexed. When an Elasticsearch cluster is congested and begins to take longer to respond than the configured request_timeout, the fluentd elasticsearch plugin will re-send the same bulk request. Since Elasticsearch can't tell its actually the same request, all documents in the request are indexed again resulting in duplicate data. In certain scenarios, this can result in essentially and infinite loop generating multiple copies of the same data.
 
-Using an _id generated by the fluentd elasticsearch plugin will communicate to Elasticsearch the uniqueness of the requests so that duplicates will be rejected or simply replace the existing records.
+Using an _hash generated by the fluentd filter elasticsearch genid plugin and fluentd elasticsearch plugin which is specified id_key as _hash in configuration will communicate to Elasticsearch the uniqueness of the requests so that duplicates will be rejected or simply replace the existing records.
```

Comment:
Perhaps rephrase a bit:

_The bundled elasticsearch_genid filter can generate a unique `_hash` key for each record, this key may be passed to the `id_key` parameter in the elasticsearch plugin to communicate to Elasticsearch the uniqueness of the requests so that duplicates will be rejected or simply replace the existing records._

Reply:
:+1: Looks good. Thank you for considering description for generate hash id section.